lib/corpus.py

```diff
@@ -338,6 +486,20 @@ def add_segment(self, segment: Segment):
         self.segments.append(segment)
 
 
+class RecordingV2(Recording):
```

Comment:
also the dump function has to be overridden because
```
        for s in self.segments:
            s.dump(out, indentation + "  ")
```
if `self.segments` is a dict now, then `s` will be the keys.